components/activity_and_insights/insights/top_reactions/top_reactions.tsx

```diff
@@ -64,18 +66,23 @@ const TopReactions = (props: WidgetHocProps) => {
         getMyTeamReactions();
     }, [getMyTeamReactions]);
 
-    const skeletonLoader = useCallback(() => {
+    const skeletonLoader = useMemo(() => {
+        const barChartHeights = [140, 178, 140, 120, 140];
         const entries = [];
+
         for (let i = 0; i < 5; i++) {
             entries.push(
                 <div
                     className='bar-chart-entry'
                     key={i}
                 >
-                    <BarChartLoader/>
-                    <CircleLoader
-                        size={20}
+                    <RectangleSkeletonLoader
+                        width='8px'
+                        height={`${barChartHeights[i]}px`}
```

Comment:
Perhaps we could make these optionally take a number and have number arguments get treated as pixel dimensions? I think React might do something similar already when using the `style` prop.

Reply:
i have added that change as well https://github.com/mattermost/mattermost-webapp/pull/11011/commits/92a6b3ade26d37e511f874b9cf0aeb4c4ad3f6b7

Reply:
Awesome, thanks